pkg/scheduling/scheduling.go

```diff
@@ -0,0 +1,38 @@
+package scheduling
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	ScheduleOtherPolicy   = "SCHED_OTHER"
+	ScheduleOtherPriority = 0
+)
+
+func GetSchedulingPolicyAndPriority(chrtCommandOutput string) (schedPolicy string, schedPriority int, err error) {
+	/*	Sample output:
+		pid 476's current scheduling policy: SCHED_OTHER
+		pid 476's current scheduling priority: 0*/
+	const (
+		policySubstr   = "current scheduling policy"
+		prioritySubstr = "current scheduling priority"
+	)
+	for _, line := range strings.Split(chrtCommandOutput, "\n") {
```

Comment:
optional: why not split with policySubstr then prioritySubstr or better yet, use a multiline regex similar to the one in parsePingResult

Reply:
regex can be skipped here as the text has only 2 lines, so we get no performance benefit of using regex.